doc/configuration/outgoing-connections.md

```diff
@@ -30,6 +30,10 @@ This allows you to create multiple dedicated pools of the same type.
 * `host` - the pool will be started for each XMPP host or host type served by MongooseIM
 * `single_host` - the pool will be started for the selected host or host type only (you must provide the name).
 
+    !!! Note
+        A global pool with name `default` is used by services that are not configured by host_type, like `service_domain_db` or `service_mongoose_system_metrics`, or by modules that don't support dynamic domains, like `mod_pubsub`.
```

Comment:
We don't define "name" here, so maybe instead of "name `default`" whe should say "tag `default`"?